web-marketplace/src/features/marketplace/CancelSellOrderFlow/hooks/useCancelSellOrderSubmit.tsx

```diff
@@ -52,6 +61,16 @@ const useCancelSellOrderSubmit = ({
 }: Props): Return => {
   const { _ } = useLingui();
   const { queryClient } = useLedger();
+  const { wallet } = useWallet();
+  const reactQueryClient = useQueryClient();
+  const { isOrganizationDashboard, organizationRole, organizationRbamAddress } =
+    useDashboardContext();
+
+  const roleConfig = organizationRole
+    ? orgRoles[organizationRole.toLowerCase() as keyof typeof orgRoles]
+    : undefined;
+  const manageSellOrdersAuthId =
+    roleConfig?.authorizations?.can_manage_sell_orders;
```

Comment:
this logic is repeated in multiple components
I had tagged you on my other PR you reviewed so you can use some util function I had already set up for that: https://github.com/regen-network/regen-web/pull/2750#discussion_r2493839299
so let's use it instead